src/xtopcom/xchain_fork/xchain_upgrade_type.h

```diff
@@ -54,7 +54,6 @@ struct xtop_chain_fork_config {
     top::optional<xfork_point_t> inner_table_tx;
     top::optional<xfork_point_t> eth_fork_point;
     top::optional<xfork_point_t> relay_fork_point;
-    top::optional<xfork_point_t> standalone_exchange_point;
```

Comment:
since fork point `standalone_exchange_point` is removed , we could do more cleanup jobs with searching: 
- [ ] `xlegacy_exchange_group_id`
- [ ] `standalone_exchange_point`

low priority anyway.

Reply:
> since fork point `standalone_exchange_point` is removed , we could do more cleanup jobs with searching:
> 
>     * [x]  `xlegacy_exchange_group_id`
> 
>     * [x]  `standalone_exchange_point`
> 
> 
> low priority anyway.

done.